.github/ISSUE_TEMPLATE.md

```diff
@@ -1,11 +1,13 @@
 First, help us help you by providing the following information when opening an issue.
 
- - [ ] Which version of Forseti Security you're using (look in `google/cloud/security/__init__.py` in your installation).
+ - [ ] Which version of Forseti Security you're using (look in `/home/ubuntu/forseti-security/google/cloud/forseti/init.py` 
+ (on the client).
```

Comment:
Should be:

[ ] Which version of Forseti Security you're using (look in `/home/ubuntu/forseti-security/google/cloud/forseti/init.py` on the client).

Reply:
Updated